requirements/requirements.txt

```diff
@@ -1,17 +1,20 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
 #    pip-compile --allow-unsafe requirements/requirements.in
 #
 certifi==2023.11.17
     # via
     #   -r requirements/requirements.in
+    #   elastic-transport
     #   requests
     #   sentry-sdk
 charset-normalizer==2.0.1
     # via requests
-elasticsearch==6.3.1
+elastic-transport==8.11.0
+    # via elasticsearch
+elasticsearch==8.11.1
```

Comment:
Is there a reason for this to be in a version from 2018?

Any issues about our actual ES server version? I don't think this was pinned to one particular version so it doesn't seem intentional.

Reply:
https://github.com/hypothesis/bouncer/pull/225#issuecomment-483421824

It looks like Rob was not happy with the fact the Bouncer's tests apparently mock Elasticsearch, and saw little value in updating the library anyway.

Bouncer's a pretty simple app so it shouldn't be too difficult to test it manually.

Could perhaps rewrite the tests to not mock Elasticsearch as well although that may be difficult since Bouncer talks directly to h's Elasticsearch. For example it's code in h, not Bouncer, that adds the documents to Elasticsearch that Bouncer reads.

Reply:
> Could perhaps rewrite the tests to not mock Elasticsearch as well although that may be difficult since Bouncer talks directly to h's Elasticsearch.

Another option is that h handles talking to ES and exposes API for bouncer to use (with suitable authentication). That's assuming that h can't just get the data from Postgres instead.